### ISSUE TITLE
Add environment variable to control which pinentry program is used

### ISSUE THREAD

**Issue by djmattyg007** [doy/rbw#39]
On Arch Linux, `/usr/bin/pinentry` looks like so:

```
#!/bin/sh

test -e /usr/lib/libgtk-x11-2.0.so.0 && exec /usr/bin/pinentry-gtk-2  "$@"

exec /usr/bin/pinentry-curses "$@"
```

If this runs on a system with a desktop environment installed, it will always try to run the graphical pinentry program. This will happen even if I'm SSH'd into the system.

GnuPG has a roundabout way of forcing use of the CLI for pinentry, by setting `GPG_TTY=$(tty)`. I'm not sure of exactly how this logic works, but it does seem to work reliably. It would be ideal if `rbw` had a similar mechanism.

**Comment by doy** [comment]
fixed in 604e09e

**Comment by djmattyg007** [comment]
Thank you so much :)
